packages/libs/error-stack/Cargo.toml

```diff
@@ -35,11 +36,12 @@ rustc_version = "0.2.3"
 
 [features]
 default = ["std", "small"]
-std = ["anyhow?/std", "once_cell?/std"]
+std = ["anyhow?/std", "once_cell?/std", "serde?/std"]
```

Comment:
For what the `std` feature of `serde` is required?

Reply:
There's no reason right now, just enabled std to keep consistent via weak dependencies but will remove.

Reply:
Just noticed that you also specified `once_cell?/std` but `hooks` implies `std`. Could you put the `features = ['std']` on `once_cell` in the dependency section directly (as drive-by)?